src/main/java/com/openlattice/shuttle/util/Parsers.java

```diff
@@ -0,0 +1,85 @@
+package com.openlattice.shuttle.util;
+
+import org.joda.time.LocalDate;
+import org.joda.time.format.DateTimeFormat;
+import org.joda.time.format.DateTimeFormatter;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.UUID;
+
+public class Parsers {
+    private static final Logger logger = LoggerFactory.getLogger( Parsers.class );
+
+    public static String getAsString( Object obj ) {
+        if ( obj != null && obj.toString() != null ) {
+            return obj.toString().trim();
+        }
+        return null;
+    }
+
+    public static Integer parseInt( Object obj ) {
+        String intStr = getAsString( obj );
+        if ( intStr != null ) {
+            try {
+                int intValue = Integer.parseInt( intStr );
```

Comment:
Why not just `return Integer.parseInt( intStr );` (same for short, long, double)?

Reply:
oh yeah whoops I thought I had fixed those, thanks